tests/testing.py

```diff
@@ -494,103 +384,123 @@ def add_arc(radius, angle):
             """
 
     # Compare the input file created for baci.
-    compare(cubit, single_precision=True, **kwargs)
+    compare(cubit, name=name, single_precision=True, **kwargs)
 
 
 @pytest.mark.parametrize(*get_pre_processor_decorator(True, True))
-def test_element_types_tet(kwargs):
-    """Create a curved solid with different tet element types."""
+def test_create_block(kwargs):
+    """
+    Test the creation of a cubit block.
+    """
+
+    # Initialize cubit.
+    cubit = CubitPy()
+    create_block(cubit, **kwargs)
+
+
+@pytest.mark.parametrize(*get_pre_processor_decorator(True, True))
+def test_create_block_numpy_arrays(kwargs):
+    """
+    Test the creation of a cubit block.
+    """
+
+    # Initialize cubit.
+    cubit = CubitPy()
+    create_block(cubit, np_arrays=True, **kwargs)
+
+
+@pytest.mark.parametrize(*get_pre_processor_decorator(True, True))
+def test_create_block_multiple(kwargs):
+    """
+    Test the creation of a cubit block multiple time to check that cubit
+    can be reset.
+    """
+
+    # Initialize cubit.
+    cubit = CubitPy()
+    create_block(cubit, **kwargs)
+
+    # Delete the old cubit object and run the function twice on the new.
+    cubit = CubitPy()
+    for _i in range(2):
+        create_block(cubit, **kwargs)
+        cubit.reset()
+
+    # Create two object and keep them in parallel.
+    cubit = CubitPy()
+    cubit_2 = CubitPy()
+    create_block(cubit, **kwargs)
+    create_block(cubit_2, **kwargs)
```

Comment:
Can you move the `test_create_block_*` functions above the `create_element_types_*` functions? This will keep stuff that belongs together close together in the code, and will also simplify the diff here.

Reply:
Arranged accordingly.